### ISSUE TITLE
Scalar types in union are not converted to Typescript

### ISSUE THREAD

**Issue by xzyfer** [dotansimha/graphql-code-generator#1473]
**Describe the bug**
Scalar types in `union` are not converted to Typescript.

**To Reproduce**
1. Go to https://graphql-code-generator.com/
2. Select `TypeScript (Server)`
3. Paste the following into the "Schema" panel

```gql
union Scalar = String | Boolean | Int
```

**Expected behavior**
```tsx
export type Scalar = string | boolean | number
```

**Schema/Documents**
```yml
generates:
  server-types.ts:
    - typescript-common
    - typescript-server
```

```gql
union Scalar = String | Boolean | Int
```

**Comment by dotansimha** [comment]
Fixed it: https://github.com/dotansimha/graphql-code-generator/pull/1483

**Comment by dotansimha** [comment]
Fix available in 1.0.0 👍
